lib/_http_outgoing.js

```diff
@@ -696,6 +706,9 @@ ObjectDefineProperty(OutgoingMessage.prototype, 'writableNeedDrain', {
 
 const crlf_buf = Buffer.from(CRLF);
 OutgoingMessage.prototype.write = function write(chunk, encoding, callback) {
+
+  this._byteCount += NumberParseInt(chunk.length);
```

Comment:
Why parse?

Reply:
I'm not sure what was I thinking hahaha 😅... I Will make the changes